docs/contributing.md

```diff
@@ -45,7 +45,7 @@ To run tests:
 To make documentation:
 
 1. **Optional:** Re-generate all `docs/langcheck*.rst` files.
-   - `sphinx-apidoc -f --no-toc --separate --module-first -t docs/_templates/ -o docs src/langcheck/ src/langcheck/stats.py`
+   - `sphinx-apidoc -f --no-toc --separate --module-first -t docs/_templates/ -o docs src/langcheck/ src/langcheck/stats.py src/langcheck/metrics/model_manager`
```

Comment:
Do we need change? I tried running both command in my env but the outputs seem to be the same

Reply:
Related: don't we need to add `docs/langcheck.metrics.model_manager.rst`?

Reply:
The page is totally empty if we include it, so I intentionally excluded it.

![CleanShot 2024-04-02 at 19 24 51@2x](https://github.com/citadel-ai/langcheck/assets/2199875/4647daa8-723e-4948-adab-406d68bbd82a)